website/package.json

```diff
@@ -6,6 +6,8 @@
     "test": "npx jest --watch"
   },
   "dependencies": {
+    "@dbt-labs/react-dbt-dag": "0.7.10",
```

Comment:
If we change what we bundle in, this should be the only thing we'd have to include